session_items.py

```diff
@@ -1,75 +1,34 @@
 from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
+import requests
+from trello_config import trello_api_key, trello_token, trello_root_url, trello_board_id, trello_todo_list_id, trello_done_list_id
+from ToDoItem import ToDoItem
 
 def get_items():
-    """
-    Fetches all saved items from the session.
-
-    Returns:
-        list: The list of saved items.
-    """
-    return session.get('items', _DEFAULT_ITEMS)
-
-
-def get_item(id):
-    """
-    Fetches the saved item with the specified ID.
-
-    Args:
-        id: The ID of the item.
-
-    Returns:
-        item: The saved item, or None if no items match the specified ID.
-    """
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
+    response = requests.get(f'{trello_root_url}/boards/{trello_board_id}/cards?key={trello_api_key}&token={trello_token}')
```

Comment:
Might be worth using the `params` optional argument here rather than use a query string